src/router/routes.js

```diff
@@ -32,10 +32,8 @@ export default [
 	{ path: '/checkout/post-purchase', component: () => import('@/pages/Checkout/PostPurchase') },
 	{ path: '/checkout/thanks', component: () => import('@/pages/Thanks/ThanksPage') },
 	{ path: '/component-demo', component: () => import('@/pages/ComponentDemo/ComponentDemo') },
-	{
-		path: '/covid19response',
-		component: () => import('@/pages/LandingPages/MGCovid19/MGCovid19'),
-	},
+	{ path: '/covid19response', component: () => import('@/pages/LandingPages/MGCovid19/MGCovid19') },
+	{ path: '/donate/support-kiva', component: () => import('@/pages/Donate/Donate') },
```

Comment:
Please update the name of the file referenced here from Donate to DonateFromMacro or something to signify this isn’t our lender facing donate page and is very specialized for a different audience.